src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -90,15 +93,14 @@ const PersonalInfoList = ({
   const isSpousePrivate = isFacStaff && keepPrivate && SpouseName !== PRIVATE_INFO;
 
   useEffect(() => {
-    async function loadMailInfo() {
+    async function loadMailboxCombination() {
       if (myProf && isStudent) {
-        //needs to be a student? ^\._./^
```

Comment:
why did you delete this comment here, I suppose it's only for student because staff/faculty do not have it

Reply:
I deleted it because I initially wasn't sure if we were going to include anyone beyond students. We are not.